### PR TITLE
Upgrate admin + report

### PR DESCRIPTION
**Pull request by lamphan95**
image

### CODE REVIEW

File: app/assets/javascripts/comment.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      $.ajax({
+        url: "/create_report",
+        type: "POST",
+        data: dataform,
+        success: function(result) {
+          $(".close").click();
+          $(".report").html(result);
+        },
+        error: function (result) {
+        $(".close").click();
```
Comment (vannam1996) [inline app/assets/javascripts/comment.js]: indent

File: app/views/categories/_create_doc_form.html.erb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<%= render "shared/error_messages", object: @category %>
+
```
Comment (vannam1996) [inline app/views/categories/_create_doc_form.html.erb]: dư line

File: app/views/comments/_report.html.erb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<%= render "shared/error_messages", object: @comment %>
+<% if user_report?  current_user.id, @document %>
+  <button type="button" class="btn btn-basic disabled"><%= t ".reported" %></button>
+<% else %>
+  <%= button_tag type: "button", class: "btn btn-danger",
+    data: {toggle: "modal", target: "#add-report"} do %>
+    <%= t ".report" %>
```
Comment (vannam1996) [inline app/views/comments/_report.html.erb]: indent

File: app/views/users/_user.html.erb [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<li>
+  <%= image_tag user.avatar.url, class: "img-circle", size: Settings.user.avatar %>
+  <%= link_to user.name, user %>
+  <% if current_user.is_admin? && !current_user?(user) %>
+    | <%= link_to t(".del"), user, method: :delete,
```
Comment (vannam1996) [inline app/views/users/_user.html.erb]: {}

**Comment by lamphan95** [comment]
ready

File: app/controllers/users_controller.rb [hunk 5]
```diff
@@ -47,9 +50,28 @@ def find_user
...
+    return if current_user.is_admin?
     @user = User.find_by id: params[:id]
     return if current_user? @user
     flash[:danger] = t "users.flash.not_correct_user"
     redirect_to root_path
   end
+
+  def update_admin user
+    if user.update_attribute :is_admin, true
+      flash[:success] = "Upgrade Admin Success"
```
Comment (dothidiemthao) [inline app/controllers/users_controller.rb]: i18n

**Comment by dothidiemthao** [comment]
commented

**Comment by lamphan95** [comment]
ready

**Pull request merged**